### PR TITLE
Xeno-candles

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Queens can grant jellies of Proliferation and Rejuvenation

# Explain why it's good for the game

> Queens can grant jellies of Proliferation, for helping capture hosts, and Rejuvenation, for assisting with either healing the hive or building it.

Capturing and supporting as warden/healer/??? are important things xenos do that are not reflected in 'slaughter', 'resilience', or 'sabotage'. This will add options its just a little qol

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
qol: Queens can grant jellies of Proliferation and Rejuvenation.
/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]
it’s just a little qol

**Comment by Ultimate-Fluff** [comment]
Tl;dr 

Proliferation = capping/hugging
Rejuvenation = healing, hive building/repair
